frontend-vue3/src/composables/api/revieweditor.ts

```diff
@@ -0,0 +1,47 @@
+import { useAxios } from "@/composables/api/axios";
+import { toRefs } from "vue";
+
+export function useReviewEditorAPI() {
+  /**
+   * Composable function for making requests to the reviews API
+   *
+   * @returns - An object containing reactive state of the request and helper functions for API requests
+   */
+
+  const baseUrl = "/reviews/";
+  const { state, get, post, put, detailUrl } = useAxios(baseUrl);
+
+  async function listInvitations(review_uuid: string) {
+    return get(`${detailUrl(review_uuid)}editor/invitations/`);
```

Comment:
Adding url building or concatenation to `useAxios` might make these methods cleaner as well